dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -1030,106 +1051,131 @@ public void Test_File_Comparison_Is_Case_Sensitive()
-        //generate secrets
-        AppSecrets secrets = builder1.withKey(appKey)
-                .withHiddenSecret("p1", "secret-1")
-                .withHiddenSecret("p2", "secret-2")
-                .build();
-        //Save it
-        api.saveSecrets(secrets, site, admin);
-        final Optional<AppSecrets> secretsOptional = api.getSecrets(appKey, site, admin);
-        Assert.assertTrue(secretsOptional.isPresent());
-        secrets = secretsOptional.get();
-
-        //AES only supports security Key of sizes of 16, 24 or 32 bytes.
-        final String password = RandomStringUtils.randomAlphanumeric(32);
-        final Key securityKey = AppsUtil.generateKey(password);
-
-        //Now that we have a valid key lets dump our selection of secrets
-        final Map<String,Set<String>> appKeysBySite = ImmutableMap.of(site.getIdentifier(), ImmutableSet.of(appKey));
-        final Path exportSecretsFile = api.exportSecrets(securityKey, false, appKeysBySite, admin);
-        assertTrue(exportSecretsFile.toFile().exists());
-
-        //Remove the secret we dumped we can re import it.
-        api.deleteSecrets(appKey, site, admin);
-
-        //import it
-        api.importSecretsAndSave(exportSecretsFile, securityKey, admin);
-
-        //verify
-        final Optional<AppSecrets> secretsOptionalPostImport = api.getSecrets(appKey, site, admin);
-        assertTrue(secretsOptionalPostImport.isPresent());
-        final AppSecrets restoredSecrets = secretsOptionalPostImport.get();
-
-        assertEquals(restoredSecrets.getKey(),secrets.getKey());
-        assertEquals(restoredSecrets.getSecrets().size(),secrets.getSecrets().size());
-        for (final Entry<String, Secret> entry : secrets.getSecrets().entrySet()) {
-            final Secret originalSecret = entry.getValue();
-            final Secret restoredSecret = restoredSecrets.getSecrets().get(entry.getKey());
-            assertTrue(originalSecret.equals(restoredSecret));
+        try {
+            api.createAppDescriptor(file, admin);
+
+            final String appKey = dataGen.getKey();
+            //generate secrets
+            AppSecrets secrets = builder1.withKey(appKey)
+                    .withHiddenSecret("p1", "secret-1")
+                    .withHiddenSecret("p2", "secret-2")
+                    .build();
+            //Save it
+            api.saveSecrets(secrets, site, admin);
+            final Optional<AppSecrets> secretsOptional = api.getSecrets(appKey, site, admin);
+            Assert.assertTrue(secretsOptional.isPresent());
+            secrets = secretsOptional.get();
+
+            //AES only supports security Key of sizes of 16, 24 or 32 bytes.
+            final String password = RandomStringUtils.randomAlphanumeric(32);
+            final Key securityKey = AppsUtil.generateKey(password);
+
+            //Now that we have a valid key lets dump our selection of secrets
+            final Map<String, Set<String>> appKeysBySite = ImmutableMap
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'ImmutableMap.of' due to existing static import 'com.google.common.collect.ImmutableMap.of'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6392590)